### ISSUE TITLE
NTR: Chlamydia trachomatis PCR, DFA, or culture positive in urine, nasal secretions or discharge (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8081]
New term request Chlamydia trachomatis PCR, DFA, or culture positive in urine, nasal secretions or discharge (hausma_130810164553): Includes NAAT; nucleic acid amplification test; DFA; direct fluorescent antibodies; DNA  UMLS: 0008149

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000731
- label: Positive chlamydia trachomatis urine nucleic acid test
- parents: [Positive urine infectious agent nucleic acid test (HP:0034889)](https://hpo.jax.org/app/browse/term/HP:0034889)
- pmid: 
- def: Detection of Chlamydia trachomatis nucleic acid in the urine by a method such as polymerase chain reaction.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8081
- synonyms:
